nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/flow/synchronization/StandardVersionedComponentSynchronizer.java

```diff
@@ -265,6 +265,8 @@ public void synchronize(final ProcessGroup group, final VersionedExternalFlow ve
                     service.migrateConfiguration(originalPropertyValues, serviceFactory);
                 } else if (extension instanceof final ReportingTaskNode task) {
                     task.migrateConfiguration(originalPropertyValues, serviceFactory);
+                } else if (extension instanceof final ParameterProviderNode parameterProvider) {
+                    parameterProvider.migrateConfiguration(originalPropertyValues, serviceFactory);
```

Comment:
@takraj This conditional never appears to be fired because Parameter Providers are never added to Created Extensions.